packages/labs/analyzer/src/lib/model.ts

```diff
@@ -176,10 +176,29 @@ export class Module {
   getExport(name: string): Declaration | Reference {
     this._ensureExportsFinalized();
     const exp = this._exportMap.get(name);
-    if (exp instanceof Reference) {
+    if (exp === undefined) {
+      throw new Error(
+        `Module ${this.sourcePath} did not contain an export named ${name}`
+      );
+    } else if (exp instanceof Reference) {
       return exp;
     } else {
-      return this.getDeclaration(name);
```

Comment:
This was a bug; it should have been using `exp` here; when it is a string, it's the name of a local declaration (which may be exported under a different name).

I didn't have good export tests; this PR adds a bunch.